src/robotlegs/bender/extensions/viewManager/impl/ContainerRegistry.ts

```diff
@@ -59,16 +59,21 @@ export class ContainerRegistry extends EventDispatcher {
      * @private
      */
     public addContainer(container: any): ContainerBinding {
-        return (this._bindingByContainer[container] =
```

Comment:
This is same as `this._bindingByContainer["[object Object]"] =` because `container.toString() === "[object Object]"`